src/index.html

```diff
@@ -11,7 +11,174 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="group">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label class="form-label" for="surname">
+            Surname:
+            <input
+              type="text"
+              name="surname"
+              id="surname"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="name" class="form-label">
+            Name:
+            <input
+              type="text"
+              name="name"
+              id="name"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
```

Comment:
@iLebo-stack few more things.
You don't really need `<div` wrappers:
- first - it makes your code bigger;
- and you don't actually style `class="form-field"` in your .css file
It applies to all `<div` across your code.

```suggestion
          <label for="name" class="form-label">
            Name:
            <input
              type="text"
              name="name"
              id="name"
              autocomplete="off"
              required
            >
          </label>
```